<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setPicClrTool</class>
 <widget class="QWidget" name="setPicClrTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>259</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>自定义色调曲线</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1,0,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>自定</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>负片（RGB）</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>彩色负片（RGB）</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>较暗（RGB）</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>较亮（RGB）</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="4,2,1">
     <item>
      <widget class="QComboBox" name="colorSetCmb">
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>RGB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>灰</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>红</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>蓝</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>绿</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>初始化</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Widget" name="widget" native="true">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>输入：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="inputEdt">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>输出：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="outputEdt">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Widget</class>
   <extends>QWidget</extends>
   <header location="global">widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
